src/commands/createEnvironment.ts

```diff
@@ -0,0 +1,40 @@
+import * as environment from '../utils/environment';
+import * as vscode from 'vscode';
+import * as shell from '../utils/shell';
+import { resolve } from 'path';
+
+export async  function buildLocalClusters() {	
+	await vscode.window.withProgress(
+		{
+		  location: vscode.ProgressLocation.Notification,
+		  title: 'Local OCM Cluster',
+		  cancellable: false,
+		},
+		async (progress) => {
+			try{
+
+				for (let i = 0; i < 3; i++) {
```

Comment:
y r we doing this loop?

Reply:
to show prgress

Reply:
but this loop doesn't do anything besides progressing the progress bar no?